src/components/learning-center/learning-searchbar.tsx

```diff
@@ -13,53 +14,126 @@ import { I18n } from "@lingui/react";
-              value={currentRefinement}
-              onChange={(event) => {
-                refine(event.currentTarget.value);
-              }}
-            />
-          </form>
-        )}
-      </I18n>
-      {!!currentRefinement && (
-        <>
-          <CustomHits />
-          <div className="search-by is-pulled-right">
-            <img
-              width="100"
-              height="20"
-              src={require("../../img/brand/algolia.svg")}
-            />
+    <FocusTrap
+      active={searchFocused}
+      focusTrapOptions={{
+        onDeactivate: () => setSearchFocused(false),
+        clickOutsideDeactivates: true,
+        returnFocusOnDeactivate: false,
+      }}
+    >
+      <div>
+        <I18n>
+          {({ i18n }) => (
+            <form
+              className="control is-flex is-justify-content-space-between is-align-items-center"
+              noValidate
+              action=""
+              role="search"
+              onSubmit={(e) => e.preventDefault()}
+            >
+              <input
+                className="input is-align-self-stretch p-4"
+                type="search"
+                onFocus={() => setSearchFocused(true)}
+                placeholder={i18n._(t`Search`)}
+                value={currentRefinement}
+                onChange={(event) => {
+                  refine(event.currentTarget.value);
+                }}
+              />
+              <img
+                src={require("../../img/search.svg")}
+                height="25"
+                width="25"
+                className="m-4"
+              />
+            </form>
+          )}
+        </I18n>
+        {searchFocused && (
+          <div className="jf-dropdown-content mt-3 py-5 px-4">
+            {!!currentRefinement ? (
+              <>
+                <CustomHits />
+                <div className="jf-search-by is-pulled-right">
+                  <img
+                    width="100"
+                    height="20"
+                    src={require("../../img/brand/algolia.svg")}
+                  />
+                </div>
+              </>
+            ) : (
+              <DropdownPlaceholder {...props} />
+            )}
           </div>
-        </>
-      )}
-    </>
+        )}
+      </div>
+    </FocusTrap>
   );
 };
 
+type DropdownPlaceholderInfo = {
+  categoryButtons: {
+    title: string;
+    slug: string;
+  }[];
+  popularArticles: {
+    slug: string;
+    title: string;
+  }[];
+};
+
+const DropdownPlaceholder: React.FC<DropdownPlaceholderInfo> = ({
+  categoryButtons,
+  popularArticles,
+}: DropdownPlaceholderInfo) => (
+  <>
+    <span className="eyebrow is-bold mb-3">Key Topics</span>
+    <div className="mt-3 mb-7">
+      {categoryButtons.map((category: any, i: number) => (
+        <a
+          href={`/learn/category/${category.slug}`}
+          className="jf-category-tag tag no-underline is-empty"
+          key={i}
+        >
+          {category.title}
+        </a>
```

Comment:
Make sure these are `<LocaleLinks>` instead of just a tags so they preserve the locale of whatever page they go to.